avi/resource_avi_cloud.go

```diff
@@ -206,6 +208,11 @@ func ResourceCloudSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Required: true,
 		},
+		"cloud_state_max_retry": {
```

Comment:
I would not put this here. Let us move this as a provider setting instead. I don't want to pollute the object model with this information.